TreeTracker/UI/Views/AddTree/AddTreeViewController.swift

```diff
@@ -61,7 +67,19 @@ class AddTreeViewController: UIViewController, AlertPresenting {
             saveTreeButton.isEnabled = false
         }
     }
-
+    @IBOutlet weak var searchGPSSignalImageView: UIImageView! {
+        didSet {
+            if gpsAccuracyLabel.accuracy == .good {
+                searchGPSSignalImageView.isHidden = true
+            } else {
+                searchGPSSignalImageView.animationImages = animatedImages(for: "gpsSearchAnimation")
+                searchGPSSignalImageView.animationDuration = 0.9
+                searchGPSSignalImageView.animationRepeatCount = 0
+                searchGPSSignalImageView.image = searchGPSSignalImageView.animationImages?.first
```

Comment:
Would probably also just use the first image directly here then it doesnt depend on the aniumation images being set:

`searchGPSSignalImageView.image = Asset.Assets.GpsSearchAnimation.gpsLoad0.image`